index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css" /> 
+    <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="background"><img src="./headline-mobile.png"></div>
+    <div class="top">
+        <div class="header">
+            <p class="logo">My Logo</p>
+        </div>
+        <div class="menuhover">
+            <button>
+                 <i class="fa-solid fa-bars"></i>
+            </button>
+        </div>
+    </div>
+    <div class="intro">
+        <p class="text">Im Gerson Higgins. Glad to see you!</p>
+        <p class="text2">I'm a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience! If you like 
+             what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+    </div>
+    <div class="icons">
+        <div class="icon1"><i class="fa-brands fa-twitter"></i></div>
+        <div class="icon2"><i class="fa-brands fa-linkedin-in"></i></div>
+        <div class="icon3"><i class="fa-regular fa-m"></i></div>
+        <div class="icon4"><i class="fa-brands fa-github"></i></div>
+        <div class="icon5"><i class="fa-regular fa-hand-peace"></i></div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**_I am marking this comment as optional because the previous reviewer did not mention it. But I strongly suggest you fix these issues as they are core requirements that the previous reviewer missed._**